tests/tests.cpp

```diff
@@ -9,23 +10,149 @@
+	EXPECT_EQ(round(initialKeyDPIMultiplier + .9), round(keyDPIMultiplier)) << "Issue 7 Testing Error 4: keyDPIMultiplier was not equal to 2\n";
+
+	// Testing resetting DPIs to default values
+	setDefaultDPI();
+	EXPECT_EQ(initialMouseDPIMultiplier, mouseDPIMultiplier) << "Issue 7 Testing Error 5: mouseDPIMultiplier was not equal to .0001\n";
+	EXPECT_EQ(initialKeyDPIMultiplier, keyDPIMultiplier) << "Issue 7 Testing Error 6: keyDPIMultiplier was not equal to .1\n";
+
+	// Testing one decrementation
+	decrementDPI();
+	EXPECT_EQ(initialMouseDPIMultiplier - .0001, mouseDPIMultiplier) << "Issue 7 Testing Error 7: mouseDPIMultiplier was not decremented by .0001\n";
+	EXPECT_EQ(initialKeyDPIMultiplier - .1, keyDPIMultiplier) << "Issue 7 Testing Error 8: keyDPIMultiplier was not decremented by .1\n";
+
+	// Testing max decrementation
+	decrementDPI();
+	EXPECT_EQ(initialMouseDPIMultiplier - .0001, mouseDPIMultiplier) << "Issue 7 Testing Error 9: mouseDPIMultiplier was not equal to 0\n";
+	EXPECT_EQ(initialKeyDPIMultiplier - .1, keyDPIMultiplier) << "Issue 7 Testing Error 10: keyDPIMultiplier was not equal to 1\n";
+}
+
+TEST(tests, issue41)
+{
+	// Testing a line of text that does not define a CRAFT keybind
+	EXPECT_EQ(isDefinedCraftKey("This is not a defined craft key", 8, 13), 0) << "Issue 41 Testing Error 1: test found string to be a defined CRAFT keybind\n";
+
+	// Testing a line of text that does define a CRAFT keybind
+	EXPECT_EQ(isDefinedCraftKey("#define CRAFT_KEY_TESTING_ISSUE_41", 8, 13), 1) << "Issue 41 Testing Error 2: test found string to not be a defined CRAFT keybind\n";
+}
+
+/// As a part of [Issue #66]: https://github.com/Team-10-But-Better/Craft/issues/66
+/// Add a test to show that a player has a measure of health
+TEST(tests, issue66health)
+{
+	initializeSurvivalStatus();
+	EXPECT_EQ(playerStatus.health, 10) << "Issue 66 Testing Health Error 1: Health was not initialized to 10.\n";
+}
+
+
+/// As a part of [Issue #66]: https://github.com/Team-10-But-Better/Craft/issues/66
+/// Add a test to show that a player can take fall damage
+TEST(tests, issue66fallDamage)
+{
+	EXPECT_EQ(playerStatus.health, 10) << "Issue 66 Testing Fall Damage Error 1: Player's health was not set to 10.\n";
+	takeFallDamage(1);
+	EXPECT_EQ(playerStatus.health, 9) << "Issue 66 Testing Fall Damage Error 2: Player's health was not decremented by 1.\n";
+	takeFallDamage(10);
+	EXPECT_EQ(playerStatus.health, 0) << "Issue 66 Testing Fall Damage Error 3: Player's health was not decremented to the minimum: 0.\n";
+}
+
+/// As a part of [Issue #72]: https://github.com/Team-10-But-Better/Craft/issues/72
+/// Add a test to show that a player's hunger can be measured
+TEST(tests, issue77hunger)
+{
+	EXPECT_EQ(playerStatus.hunger, 10) << "Issue 72 Testing Hunger Error 1: Player's hunger was not set to 10.\n";
+	int time = 0;
+	for (int i = 0; i <= 1001; i++)
+	{
+		time++;
+	}
```

Comment:
if you need to have time variable value as 1001, why have you initialized it to 0 and running For loop for just incrementing its value to 1001?

you can simply set time variable value as 1001 this will reduce complexity of program.